Add dolar tj and converter

pabloferrari opened the pull request:
Added dollar TJ quote
![image](https://user-images.githubusercontent.com/10764358/161167841-61e97012-6af7-44f9-b3d5-cff63e8dd4b7.png)

Converter: 

USD to ARS -> `dolar :amount`
ARS to USD -> `dolar :amount --to-usd`

![image](https://user-images.githubusercontent.com/10764358/161168049-2418907e-c4a7-426b-b270-b19811bc4b91.png)

pabloferrari committed:
add dolar tj and conversor

pabloferrari committed:
fix

martincarrera commented:
Hey @pabloferrari, sorry I didn't see your PR before. 

Can you extract the `1.65` to a constant at the top of the file, and verify if that number is still correct?

Thanks for taking the time to add this!

pabloferrari committed:
code review and result changes

pabloferrari committed:
conversion rate details

pabloferrari commented:
Hey @martincarrera, I just updated the PR. 

- extracted CC conversion rate
- changes in results and conversion

<img width="607" alt="Screenshot 2022-12-28 at 12 23 14" src="https://user-images.githubusercontent.com/10764358/209834189-c7c9d665-d91f-4f61-b9e7-7969c4e76f11.png">
<img width="179" alt="Screenshot 2022-12-28 at 12 21 59" src="https://user-images.githubusercontent.com/10764358/209834205-683e51e9-c234-4a4a-b848-24dccd0aae79.png">

pabloferrari committed:
fix
